modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,52 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which you can configure by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+.Prerequisites
+
+* You must have the {ServerlessOperatorName} and Knative Serving installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Knative service that contains the label or annotation that you want to propagate to the {product-title} route.
+** Create a service by using YAML:
```

Comment:
Could be "To create a service by using YAML:"

Reply:
Could it make sense to include a link to resource creation docs as an additional resource following this module?

Reply:
I think it's OK without this tbh, we have a ton of examples of creating resources in serverless docs and don't reference it every time, so I think if we were going to do this it would be better to make the decision to update it across the board. I will leave it off for now since it's a bit out of scope for this PR.